eslint.config.ts

```diff
@@ -62,6 +67,10 @@ export default defineConfig(
 			"no-useless-rename": "error",
 			"object-shorthand": "error",
 			"operator-assignment": "error",
+			"unicorn/no-array-reverse": "off", // Overly strict
+			"unicorn/no-array-sort": "off", // Overly strict
+			"unicorn/prefer-string-raw": "off",
+			"unicorn/prefer-string-replace-all": "off",
```

Comment:
[Style] Nit: to not have to write the same thing multiple times, and aligning with previous "groups of rules" comments:

```suggestion
			// These on-by-default rules are too strict for our preferences
			"unicorn/no-array-reverse": "off",
			"unicorn/no-array-sort": "off",
			"unicorn/prefer-string-raw": "off",
			"unicorn/prefer-string-replace-all": "off",
```